src/main/java/ua/dymohlo/jdbcpractic/JDBCMethods.java

```diff
@@ -0,0 +1,128 @@
+package ua.dymohlo.jdbcpractic;
+
+import java.sql.*;
+
+public class JDBCMethods {
+    public static void insertIntoTableContact() {
+        try (Connection connection = ConnectionManager.getConnection()) {
+            String newMailContact = "INSERT INTO alevelgraduates.contact(contact, contact_type) VALUES ('snowSecond@gmail.com', 'mail')";
```

Comment:
Here you have hardcoded values(snowSecond@gmail.com, mail). You should move such values to method parameters. Then pass these values to the method. Because, what if we want insert other values? We should to create new method then..

insertIntoTableContact(String email, String contactType) { // here you work with email and contactType}

insertIntoTableContact(snowSecond@gmail.com, mail)